src/jvm/test/resources/expected_logs/thread_creation_representation_test.txt

```diff
@@ -0,0 +1,65 @@
+|   thread(name = "thread2") { ... } : Thread#3 at ThreadCreationRepresentationTest.operation(ThreadCreationRepresentationTest.kt:24)               |                                                                                                                                                                                            |                                                                                                                                                                                            |                                                                                                                                                                                            |
+|     Thread#3.start() at ThreadsKt.thread(Thread.kt:42)                                                                                            |                                                                                                                                                                                            |                                                                                                                                                                                            |                                                                                                                                                                                            |
+|   thread(name = "thread3", priority = 8) { ... } : Thread#4 at ThreadCreationRepresentationTest.operation(ThreadCreationRepresentationTest.kt:26) |                                                                                                                                                                                            |                                                                                                                                                                                            |                                                                                                                                                                                            |
+|     Thread#4.start() at ThreadsKt.thread(Thread.kt:42)                                                                                            |                                                                                                                                                                                            |                                                                                                                                                                                            |                                                                                                                                                                                            |
```

Comment:
Let's remove `Thread.start()` calls when Kotlin's `thread` is called -- we do not need to show the standard library implementation details